test/unit/mocks/entities.js

```diff
@@ -99,6 +100,22 @@ const appUploadMock = {
   }),
 }
 
+const bulkActionMock = {
```

Comment:
I would either export one mock for each bulk action type or use the non-versioned link for this since 2 of the 3 bulk action payloads in use today don't use a versioned link

Reply:
I added one for publish (separate from the others).